datastore/tests/unit/test_key.py

```diff
@@ -430,6 +435,15 @@ def test_from_legacy_urlsafe_needs_padding(self):
         self.assertIsNone(key.namespace)
         self.assertEqual(key.flat_path, self._URLSAFE_FLAT_PATH2)
 
+    def test_from_legacy_urlsafe_with_location_prefix(self):
```

Comment:
@egalpin This test thinks it's a `from_legacy_urlsafe` test, but it is testing `to_legacy_urlsafe` (as it should be).

Do you mind changing the name and moving it up into the section just before `def test_from_legacy_urlsafe(self):` (line 416)?

If not, I can take the wheel, I really appreciate you taking the time to write the test.